test/dates/adjusters.jl

```diff
@@ -219,8 +219,6 @@ dt = Dates.Date(2014,5,21)
 @test Dates.tonext(dt,Dates.Sun) == Dates.Date(2014,5,25)
 @test Dates.tonext(dt,Dates.Mon) == Dates.Date(2014,5,26)
 @test Dates.tonext(dt,Dates.Tue) == Dates.Date(2014,5,27)
-# No dayofweek function for out of range values
-#@test_throws KeyError Dates.tonext(dt,8)
 
```

Comment:
A test with

```julia
@test Dates.firstdayofweek(Dates.DateTime(2013,12,24), firstday=Monday) == Dates.DateTime(2013,12,23)
@test Dates.firstdayofweek(Dates.DateTime(2013,12,24), firstday=Sunday) == Dates.DateTime(2013,12,22)
```

could be a good idea

and a similar test with `lastdayofweek`

Something like

````julia
@test Dates.lastdayofweek(Dates.DateTime(2013,12,24), firstday=Monday) == Dates.DateTime(2013,12,29)
@test Dates.lastdayofweek(Dates.DateTime(2013,12,24), firstday=Sunday) == Dates.DateTime(2013,12,28)
```